autopush/settings.py

```diff
@@ -160,6 +160,9 @@ def __init__(self,
         # db objects
         self.current_msg_month = make_rotating_tablename(self._message_prefix)
         self.current_month = datetime.date.today().month
+        # a range of months that we consider a message to be "valid"
+        self.valid_months = [self.current_msg_month,
+                             make_rotating_tablename(self._message_prefix, -1)]
```

Comment:
Technically 2 months in the past are valid. Otherwise our max message retention is only ~ 27 days. Also, if this value is not auto-updated, then the valid months will not be the current month when the day flips over to the new month.